modules/gdnative/gdnative_api.json

```diff
@@ -507,13 +507,6 @@
           ["const godot_color *", "p_self"]
         ]
       },
-      {
-        "name": "godot_color_gray",
-        "return_type": "godot_real",
-        "arguments": [
-          ["const godot_color *", "p_self"]
-        ]
-      },
```

Comment:
This would break compatibility with older versions of GDNative libraries, not sure if we want to do that.

Reply:
Unless we want to leave a method that does nothing, it's probably necessary. Breaking ABI compatibility should be fine for 4.0.